cr17/nodes/cr17/obstacle_mapper.py

```diff
@@ -1,15 +1,38 @@
-#!/usr/bin/env python
+#!/usr/bin/env python22
```

Comment:
I don't think there is a python22 yet, unless it is the 2.2 version of python, in which case python needs to work on its naming convention.